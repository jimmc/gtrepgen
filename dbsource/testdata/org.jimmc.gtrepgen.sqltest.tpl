Report for companyid {{.}}
Company data:
{{with row "select name from company where id = ?" .}}
    Name: {{.name}}
{{end}}
People in that company:
{{range rows "select firstname, lastname from person where companyid = ?" .}}
    First name: {{.firstname}}
    Last name:  {{.lastname}}
{{end}}
