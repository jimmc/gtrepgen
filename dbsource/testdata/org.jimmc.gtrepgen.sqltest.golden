Report for companyid x
Company data:

    Name: Example, Inc.

People in that company:

    First name: John
    Last name:  Doe

    First name: Jane
    Last name:  Doe

